harvard-university-of-worcester.csl

```diff
@@ -0,0 +1,376 @@
+  <info>
+    <title>Harvard - University of Worcester</title>
+    <id>http://www.zotero.org/styles/harvard-university-of-worcester</id>
+    <link rel="self" href="http://www.zotero.org/styles/harvard-university-of-worcester"/>
+    <link rel="documentation" href="http://www.worc.ac.uk/ils/documents/Harvard_referencing"/>
+    <author>
+      <name>Christopher Hill</name>
+    </author>
+    <contributor>
+      <name>Christopher Hill</name>
+      <uri>http://www.mendeley.com/profiles/christopher-hill7/</uri>
+    </contributor>
+    <contributor>
+      <name>Sarah Pittaway</name>
+    </contributor>
+    <category citation-format="author-date"/>
+    <summary>Harvard author-date style edited for the University of Worcester</summary>
+    <updated>2015-08-07T14:39:21+00:00</updated>
+	<rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="editor">
+    <names delimiter=", " variable="editor">
+      <name delimiter=", " and="symbol"/>
+      <label form="short" suffix=")" prefix=" ("/>
+    </names>
+  </macro>
+  <macro name="anon">
+    <text form="short" text-case="capitalize-first" term="anonymous" strip-periods="true"/>
+  </macro>
+  <macro name="author">
+    <names variable="author">
+      <name delimiter=", " initialize-with="." and="symbol" delimiter-precedes-last="never" sort-separator=", " name-as-sort-order="all"/>
+      <label form="short" prefix=" "/>
+      <substitute>
+        <text macro="anon"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="author-short">
+    <names variable="author">
+      <name form="short" delimiter=", " and="symbol" delimiter-precedes-last="never"/>
+      <et-al font-style="normal"/>
+      <substitute>
+        <names variable="translator"/>
+        <text macro="anon"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="available-from-access">
+    <choose>
+      <if match="any" variable="URL DOI">
+        <group>
+          <text value="Available from: "/>
+          <choose>
+            <if variable="DOI">
+              <text variable="DOI" prefix="doi:"/>
+            </if>
+            <else>
+              <text variable="URL"/>
+            </else>
+          </choose>
+          <group suffix="]" prefix=" [">
+            <text suffix=": " text-case="capitalize-first" term="accessed"/>
+            <date variable="accessed">
+              <date-part name="day" suffix=" "/>
+              <date-part name="month" suffix=" "/>
+              <date-part name="year"/>
+            </date>
+          </group>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="italicised-title">
+    <text variable="title" font-style="italic"/>
+  </macro>
+  <macro name="standard-title">
+    <text variable="title"/>
+  </macro>
+  <macro name="publisher-place">
```

Comment:
Don't solve this with text values -- that's awfully fragile. 
Something like

```
<group delimter=", " suffix=".">
  <text variable="publisher-place"/>
  <text variable="publisher/>
  <text variable="page"/>
</group>
```

Will do the same more elegantly and more robustly.